app/onboarding/review/styles.ts

```diff
@@ -0,0 +1,73 @@
+'use client';
+
+import NextImage from 'next/image';
+import styled from 'styled-components';
+import { Sans } from '@/styles/fonts';
+import COLORS from '../../../styles/colors';
+
+export const Image = styled(NextImage)`
+  width: 20px;
+  height: 20px;
+  margin-top: 1rem;
+`;
+
+export const Title = styled.h1`
+  font-size: 24px;
+  text-align: start;
+  color: ${COLORS.gray11};
+  margin-top: 0;
+  font-weight: 500;
+  margin-bottom: 28px;
+`;
+
+export const ReviewContainer = styled.main`
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+  display: flex;
+  padding: 32px 32px 32px 32px;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  background: #fefdfc;
+  border-radius: 16px;
+  box-shadow: 4px 4px 4px 0px rgba(0, 0, 0, 0.15);
+`;
+
+export const SmallText = styled.text`
```

Comment:
same thing here, try to use global text! this will also be `SMALL`

here's a good example: `<H6 $fontWeight="500">`